dotcom-rendering/src/client/poorPerformanceMonitoring.ts

```diff
@@ -46,12 +46,21 @@ const isTimeToFirstByteAboveThreshold = (threshold = 1200) => {
 		const [nav] = window.performance.getEntriesByType('navigation');
 		if (!nav) return true;
 
-		logPerformanceInfo('navigation', {
-			domContentLoadedEventEnd: nav.domContentLoadedEventEnd,
-			type: nav.type,
-			responseEnd: nav.responseEnd,
-		});
-		return nav.responseStart - nav.startTime > threshold;
+		const isPerformanceNavigationTiming = (
+			navigationEvent: PerformanceEntry | undefined,
+		): navigationEvent is PerformanceNavigationTiming => {
+			return navigationEvent?.entryType === 'navigation';
+		};
+
+		if (isPerformanceNavigationTiming(nav)) {
+			logPerformanceInfo('navigation', {
+				domContentLoadedEventEnd: nav.domContentLoadedEventEnd,
+				type: nav.type,
+				responseEnd: nav.responseEnd,
+			});
+			return nav.responseStart - nav.startTime > threshold;
+		}
+		return false;
```

Comment:
Could you explain why this file is touched? I do not see any relation with Google Analytics in here and we favour small, focussed PRs.

Reply:
Ravi and I had to make that change in Commercial because some of the deleted GA files were importing types by default (There is a name for it but can't remember it!) so there was an error if we didn't make that change. I found the same problem in DCR so that is way I implemented the same solution in here. There is no better person to explain that than @arelra.

Reply:
`@types/google.analytics` had an ambient type which overrode the lib.dom type for `PerformanceEntry`. Removing `@types/google.analytics` meant we had to narrow the type to `PerformanceNavigationTiming` explicitly.

Reply:
I would be a little wary of adding type predicates like this without a test, as they are no safer than type casting. Would it not be sufficient to check `nav instanceof PerformanceNavigationTiming`?

Reply:
Yes Max it is sufficient so I made the change 018e5c3ee63a30f54d883f8ee0dee162ef5dc935. Thanks!